release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -103,6 +103,11 @@ Warning: short name "ex" could also match lower priority resource examples.test.
 [id="ocp-4-16-networking"]
 === Networking
 
+[id="ocp-4-16-networking-connectivity-pod-placement"]
+==== Configurable placement of network connectivity check pods
```

Comment:
```suggestion
==== Configuring pod placement to check network connectivity 
```

Reply:
IMO, the heading can be updated for clearer context

Reply:
If you don't want to use gerund or singular (pod) then probably something like this: "Configure the placement of pods to check network connectivity"